bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=argocd-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.9.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.10.0+git
```

Comment:
why does this say v1.10 when the sdk upgraded to v1.11 ?

Reply:
Those values are generated by the operator-sdk.  I don't know why they seem to be behind, but the original `operator-sdk-v1.9.0+git` was generated by version 1.10.0 of the operator-sdk, and I've seen it with other versions of the sdk, so it's not specific to the 1.11.0 version.